lib/sqlalchemy/sql/selectable.py

```diff
@@ -3660,11 +3687,12 @@ def set_label_style(
         self, label_style: SelectLabelStyle
     ) -> SelectStatementGrouping:
         return SelectStatementGrouping(
```

Comment:
**mike bayer** (zzzeek) wrote:

remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325